src/CosmosDB/CosmosDB/CosmosDB.generated.format.ps1xml

```diff
@@ -69,5 +69,197 @@
         </ListEntries>
       </ListControl>
     </View>
+    <View>
+      <Name>Microsoft.Azure.Management.CosmosDB.Models.PSRestorableMongodbCollectionGetResult</Name>
+      <ViewSelectedBy>
+        <TypeName>Microsoft.Azure.Management.CosmosDB.Models.PSRestorableMongodbCollectionGetResult</TypeName>
+      </ViewSelectedBy>
+      <ListControl>
+        <ListEntries>
+          <ListEntry>
+            <ListItems>
+              <ListItem>
+                <PropertyName>Id</PropertyName>
+                <Label>Id</Label>
+              </ListItem>
+              <ListItem>
+                <PropertyName>Name</PropertyName>
+                <Label>Name</Label>
+              </ListItem>
+              <ListItem>
+                <PropertyName>Type</PropertyName>
+                <Label>Type</Label>
+              </ListItem>
+              <ListItem>
+                <PropertyName>_rid</PropertyName>
+                <Label>_rid</Label>
```

Comment:
Looks like _rid is private property, is there any public alternative for _rid? please replace it.
BTW, label can be different with PropertyName, I suggest ResourceId if rid means `resource id`.

Reply:
Updated to ResourceId